src/analysis/special_functions/pow.lean

```diff
@@ -752,6 +758,51 @@ lemma deriv_within_sqrt (hf : differentiable_within_at ℝ f s x) (hx : f x ≠
 
 end differentiability
 
+section limits
+open real filter
+
+/-- The function `x^y` tends to `+∞` at `+∞` for any positive real `y` -/
+lemma tendsto_rpow_at_top {y : ℝ} (hy : 0 < y) : tendsto (λ x : ℝ, x ^ y) at_top at_top :=
+begin
+  rw tendsto_at_top_at_top,
+  intro b,
+  use (max b 0) ^ (1/y),
+  intros x hx,
+  exact le_of_max_le_left
+    (by { convert rpow_le_rpow (rpow_nonneg_of_nonneg (le_max_right b 0) (1/y)) hx (le_of_lt hy),
+      rw [← rpow_mul (le_max_right b 0), (eq_div_iff (ne_of_gt hy)).mp rfl, rpow_one] }),
+end
+
+/-- The function `x^(-y)` tends to `0` at `+∞` for any positive real `y` -/
+lemma tendsto_rpow_neg_at_top {y : ℝ} (hy : 0 < y) : tendsto (λ x : ℝ, x ^ (-y)) at_top (𝓝 0) :=
+tendsto.congr' (eventually_eq_of_mem (Ioi_mem_at_top 0) (λ x hx, (rpow_neg (le_of_lt hx) y).symm))
+  (tendsto.inv_tendsto_at_top (tendsto_rpow_at_top hy))
+
+/-- The function `x ^ (a / (b * x + c))` tends to `1` at `+∞`, for any real numbers `a`, `b`, and
+`c` such that `b` is nonzero. -/
+lemma tendsto_rpow_div_mul_add (a b c : ℝ) (hb : 0 ≠ b) :
+  tendsto (λ x, x ^ (a / (b*x+c))) at_top (𝓝 1) :=
+begin
+  refine tendsto.congr' _ ((tendsto_exp_nhds_0_nhds_1.comp
+    (by simpa only [mul_zero, pow_one] using ((@tendsto_const_nhds _ _ _ a _).mul
+      (tendsto_div_pow_mul_exp_add_at_top b c 1 hb (by norm_num))))).comp (tendsto_log_at_top)),
+  apply eventually_eq_of_mem (Ioi_mem_at_top (0:ℝ)),
+  intros x hx,
+  simp only [set.mem_Ioi, function.comp_app] at hx ⊢,
+  rw [exp_log hx, ← exp_log (rpow_pos_of_pos hx (a / (b * x + c))), log_rpow hx (a / (b * x + c))],
+  field_simp,
+end
+
+/-- Special case of `tendsto_rpow_div_mul_add` with `a = 1`, `b = 1`, and `c = 0` -/
```

Comment:
An important use of docstrings is that they show up in autocompletion tooltips. So, they should be understandable without any context. Can you change the docstring to make it self-contained? Same thing for the next one.